src/main/java/balckjack/domain/Players.java

```diff
@@ -40,10 +42,10 @@ private boolean isFormat(String names) {
     }
 
     private void validateDuplicateName(List<String> names) {
-        List<String> duplicateNames = names.stream()
-            .filter(i -> Collections.frequency(names, i) > 1)
-            .distinct()
-            .collect(Collectors.toList());
+        Set<String> distinctNames = new HashSet<>();
+        Set<String> duplicateNames = names.stream()
+            .filter(name -> !distinctNames.add(name))
+            .collect(Collectors.toSet());
         if (duplicateNames.size() > 0) {
             throw new IllegalArgumentException(
                 String.format("중복되는 이름이 존재합니다. : %s", String.join(", ", duplicateNames)));
```

Comment:
지금도 잘 해주셨는데 HashSet 생성자의 인자로 컬렉션을 넘겨서 생성할 수 있답니다 🙂

```suggestion
        if (names.size() != new HashSet<>(names).size()) {
            throw new IllegalArgumentException();
        }
```